app/src/main/java/de/westnordost/streetcomplete/quests/access_point_ref/AddAccessPointRef.kt

```diff
@@ -0,0 +1,41 @@
+package de.westnordost.streetcomplete.quests.access_point_ref
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddAccessPointRef : OsmFilterQuestType<AccessPointRefAnswer>() {
+
+    override val elementFilter = """
+        nodes with
+        (
+          highway = emergency_access_point
+          or emergency = access_point
+        )
+        and !name and !ref and noref != yes and ref:signed != no and !~"ref:.*"
+    """
+
+    override val changesetComment = "Determine emergency access point refs"
+    override val wikiLink = "Tag:emergency=access_point"
+    override val icon = R.drawable.ic_quest_access_point
+    override val achievements = listOf(EditTypeAchievement.LIFESAVER, EditTypeAchievement.RARE)
+    override val isDeleteElementEnabled = true
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_accessPointRef_title
+
+    override fun createForm() = AddAccessPointRefForm()
+
+    override fun applyAnswerTo(answer: AccessPointRefAnswer, tags: Tags, timestampEdited: Long) {
+        when (answer) {
+            is NoAccessPointRef -> tags["noref"] = "yes"
+            is AccessPointRef ->   tags["ref"] = answer.ref
+            is IsAssemblyPointAnswer -> {
```

Comment:
Some of these can have ref's too, e.g. letters, like everyone from one building or team go to A, another B etc:
https://www.manutan.co.uk/img/S/GRP/ST/AIG3603434.jpg

Although not all, and it's probably less common/only bigger sites.